MultiSource/Benchmarks/Olden/treeadd/node.c

```diff
@@ -124,7 +124,7 @@ int TreeAdd (tree_t *t)
 #else
     int leftval;
     int rightval;
-    tree_t *tleft, *tright;
+    Tree tleft = NULL, tright = NULL;
```

Comment:
I think it's worth assuming that the contents of headers we can't find only provide additional definitions, but these will still work with ours. Until I can find a copy of "future-cell.h", I'm not that bothered, as we'll never compile in a way that this file is needed, but I do want the two halves of the code to line up.